docs/source/deployment/snippets/14_run_smoke_tests.partial.md

```diff
@@ -10,4 +10,4 @@ They are automatically uploaded to the SRD during the deployment step.
     - if any of the tests fail, check the `README.md` in this folder for help in diagnosing the issues
 - Copy `tests/test_jupyter.ipynb` to your home directory
     - activate each of the available Python versions in turn
-    - run `jupyter notebook` in each case and check that you can run the notebook and that all versions and paths match throughout. See {ref}`Available Python Versions <available_python_and_r_versions>`.
+    - run `jupyter notebook` in each case and check that you can run the notebook and that all versions and paths match throughout. See [Available Python and R versions](<project:../roles/researcher/user_guide_guacamole.md#available-python-and-r-versions>)
```

Comment:
```suggestion
    - run `jupyter notebook` in each case and check that you can run the notebook and that all versions and paths match throughout. See [Available Python and R versions](/roles/researcher/user_guide_guacamole.md#available-python-and-r-versions)
```

I think the autolink `<>` is redundant as we have explicit text.
I think not using `project:` is clearer as it is the default anyway.

(I suppose the `..` is ignored :confused:)

Reply:
According to the myst docs, relative or absolute links, like the one below, _should_ work:

```
[](../roles/researcher/user_guide_guacamole.md#available-python-and-r-versions
```

But they don't seem to. The link it generates ends up pointing to the page the document it was included on, but looking for an anchor with the specified filepath -

like 

```
https://data-safe-haven.readthedocs.io/en/latest/deployment/deploy_sre_apache_guacamole.html#../roles/researcher/user_guide_guacamole.md#available-python-and-r-versions
```

After a lot of attempts, the autolink style was the only way I could get it to correctly generate the link.

Reply:
I did indeed miss something, and now it works (sort-of) correctly, so must have mixed something up 🙈 Despite me telling it to link to the `guacamole` user guide, it links to the `msrds` user guide, but that'll do for now, I think